### DISCUSSION TITLE
Advice on how to log in to multiple accounts with go_router?

### DISCUSSION THREAD

**Discussion by jacob-emery** [csells/go_router#157] [General]
Disclaimer: I am a junior developer.
I am considering `go_router` for managing routes. I would like to make an app that allows the end users to log in to multiple accounts (similar to how Instagram manages this). Since I didn't see this feature mentioned in the docs, I wonder if it is something currently supported.
What would be the best practice using this package?

**Comment by csells** [comment]
You should be able to handle multiple logins with go_router. The lack of a specific sample isn't an indication of something like that not being supported.

**Reply by jacob-emery** [reply]
Thank you very much for your quick reply! I will definitely try to do so.
